lib/kubernetes-deploy/kubernetes_resource/cloudsql.rb

```diff
@@ -0,0 +1,66 @@
+module KubernetesDeploy
+  class Cloudsql < KubernetesResource
+    TIMEOUT = 30.seconds
+
+    def initialize(name, namespace, file)
+      @name, @namespace, @file = name, namespace, file
+    end
+
+    def sync
+      _, st = run_kubectl("get", type, @name)
+      @found = st.success?
+      @status = true
+
+
+
+      log_status
+    end
+
+    def deploy_succeeded?
+      cloudsql_proxy_deployment_exists? && mysql_service_exists?
+    end
+
+    def deploy_failed?
+      !cloudsql_proxy_deployment_exists? || !mysql_service_exists?
```

Comment:
Only via logs sent to splunk. There is no Status field on ThirdPartyResources yet. If and when there is one, we could report it.